reward/supply_manager.go

```diff
@@ -143,9 +144,9 @@ func (sm *supplyManager) GetAccReward(num uint64) (*database.AccReward, error) {
 		return nil, errNoAccReward
```

Comment:
Currently this branch isn't effective in production since usually it has cache for the recent blocks. But if caches are gone, it may return error even if it can calculate the latest `accReward` thanks to this PR (re-accumulate from `lastNum` to `num`). Do you think it'll be still effective after this PR?

Reply:
I think we don't need to consider unessential case. Sorry for the hussle.

Reply:
Sounds fair. I think I intended block unwinding, but unwinding can be done by DeleteAccReward instead.

Reply:
Yes the branch is harmless. Let's keep it this way.